rb/spec/integration/selenium/webdriver/element_spec.rb

```diff
@@ -94,6 +94,11 @@ module WebDriver
         expect(driver.find_element(id: 'withText').attribute('nonexistent')).to be_nil
       end
 
+      it 'should get property value' do
```

Comment:
This needs to add a guard. Looks like [Edge hasn't implemented it, yet](https://developer.microsoft.com/en-us/microsoft-edge/platform/documentation/webdriver-commands/), so:
```
compliant_on browser: :firefox do 
```